exonum-java-binding/time-oracle/README.md

```diff
@@ -32,16 +32,17 @@ Before that, the method will return a result with an empty value. Validator node
 transactions after the commit of each block.
 
 ```java
-private final Node node;
-
-public Optional<ZonedDateTime> getTime() {
-  return node.withSnapshot(s -> {
-    TimeSchema timeOracle = TimeSchema.newInstance(s);
-    EntryIndexProxy<ZonedDateTime> currentTime = timeOracle.getTime();
-    return toOptional(currentTime);
-  });
-}
+  public Optional<ZonedDateTime> getTime(View view) {
```

Comment:
> Below is an example of a service method

Do clients normally pass views or do services request it from the `Node`? One could definitely have such a method in a service, but to get a view the clients will have to use a node.